contracts/base/DomainOwnershipManager.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../errors/DomainOwnershipErrors.sol";
+import "../events/DomainOwnershipEvents.sol";
+
+/// @title Domain Ownership Manager
+/// @dev This contract provides service of domain registration
+contract DomainOwnershipManager is DomainOwnershipErrors, DomainOwnershipEvents {
+
+    address public owner;
+
+    uint256 public registrationFee = 0.00001 ether;
+
+    mapping(string => address) public domainToOwner;
+
+    mapping(address => string[]) public ownerToDomains;
+
+    constructor() {
+        owner = msg.sender;
+    }
+
+    /// @dev Checks if the caller is contract owner
+    modifier onlyOwner() {
```

Comment:
line 24 
as good practice, to set modifier before the constructor function
// - 1 point